<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>248</width>
      <height>183</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Time</string>
    </property>
    <widget class="QLabel" name="clock">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="alarmVolume">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm Volume</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkWeekDays">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DaysOfWeek</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkWeekEnd">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DaysOfWeekEnd</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="weekDays_edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="weekEndDays_edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkUserDateTime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>User Time and Date</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="userTimeAndDate_edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="alarm1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alram1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alarm2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alarm3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alarm4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alarm5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSound">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Sound</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="soundPathLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="weatherInfoBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Weather</string>
    </property>
    <widget class="QLabel" name="weatherInfo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cityInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperatureInfo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="weatherInfo_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="weatherButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Weather</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
